facia/app/services/dotcomrendering/FaciaPicker.scala

```diff
@@ -93,13 +80,8 @@ object FrontChecks {
   def isNotPaidContent(faciaPage: PressedPage): Boolean = {
     // We don't support paid content
     // See: https://github.com/guardian/dotcom-rendering/issues/5945
-    !faciaPage.collections.exists(_.curated.exists(card => card.properties.isPaidFor))
-  }
-
-  def hasNoPaidForCards(faciaPage: PressedPage): Boolean = {
-    // We don't support labs containers
     // See: https://github.com/guardian/dotcom-rendering/issues/5150
-    !faciaPage.collections.exists(collection => collection.curated.exists(card => card.isPaidFor))
```

Comment:
`card.isPaidFor` is just a wrapper for `card.properties.isPaidFor` https://github.com/guardian/frontend/blob/c7ddab98017f9c97e2fa2a3715de526bd02d576b/common/app/model/pressedContent.scala#L28